src/test/ui/on-unimplemented/bad-annotation.stderr

```diff
@@ -18,5 +18,59 @@ error[E0231]: only named substitution parameters are allowed
 35 | #[rustc_on_unimplemented = "Unimplemented trait error on `{Self}` with params `<{A},{B},{}>`"]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-error: aborting due to 3 previous errors
+error[E0232]: this attribute must have a valid value
+  --> $DIR/bad-annotation.rs:40:26
+   |
+40 | #[rustc_on_unimplemented(lorem="")]
+   |                          ^^^^^^^^ expected value here
+   |
+   = note: eg `#[rustc_on_unimplemented = "foo"]`
+
+error[E0232]: this attribute must have a valid value
+  --> $DIR/bad-annotation.rs:43:26
+   |
+43 | #[rustc_on_unimplemented(lorem(ipsum(dolor)))]
+   |                          ^^^^^^^^^^^^^^^^^^^ expected value here
+   |
+   = note: eg `#[rustc_on_unimplemented = "foo"]`
+
+error[E0232]: this attribute must have a valid value
+  --> $DIR/bad-annotation.rs:46:39
+   |
+46 | #[rustc_on_unimplemented(message="x", message="y")]
+   |                                       ^^^^^^^^^^^ expected value here
```

Comment:
Should we handle duplicated values with a special label pointing at the first one encountered?

Reply:
I didn't bother with precise error reporting here - this is an unstable feature that is going to be revised.